sqs/src/main/scala/akka/stream/alpakka/sqs/SqsSourceSettings.scala

```diff
@@ -25,6 +25,19 @@ object SqsSourceSettings {
                       attributeNames.asScala,
                       messageAttributeNames.asScala)
 
+  def create(waitTimeSeconds: Int,
+             maxBufferSize: Int,
+             maxBatchSize: Int,
+             attributeNames: util.List[AttributeName],
+             messageAttributeNames: util.List[MessageAttributeName],
+             closeOnEmptyReceive: Boolean): SqsSourceSettings =
```

Comment:
SInce this is the builder for just such a settings class (`SqsSourceSettings`), I am not sure if there would be a benefit. There is default parameters on the scala side, and shorter overloads for java users, after all.

Reply:
Aah, sorry, it's morning and I'm not quite awake yet so I read it as the constructor of the stage. NVM that comment. :)